### ISSUE TITLE
DNS and Connect timings

### ISSUE THREAD

**Issue by gnublin** [macbre/phantomas#477]
Hello,

Do you have an option to get DNS and Connect time ? (like performance.timing value)

Regards,
Gnublin.

**Comment by macbre** [comment]
@gnublin, we're blocked on https://github.com/ariya/phantomjs/issues/10245

**Comment by macbre** [comment]
[SlimerJS supports](https://gist.github.com/macbre/03efa4479ab80dca4437#slimerjs) [Resource Timing API](http://www.stevesouders.com/blog/2014/08/21/resource-timing-practical-tips/) which exposes information such as connection and DNS lookup times.

**Comment by macbre** [comment]
Introduce `performanceTiming` module with the following metrics:

* `timingDNS`
* `timingConnect`

```js
// Navigation Timing
var t = performance.timing,
    pageloadtime = t.loadEventStart - t.navigationStart,
    dns = t.domainLookupEnd - t.domainLookupStart,
    tcp = t.connectEnd - t.connectStart,
    ttfb = t.responseStart - t.navigationStart;
```

http://www.stevesouders.com/blog/2014/08/21/resource-timing-practical-tips/

**Comment by macbre** [comment]
Implemented (see #567), however I get strange results in both SlimerJS and PhantomJS 2.0. Times for DNS and HTTP connection are always zero and page load time is pretty small compared to the values of the other metrics.

```
$ ./bin/phantomas.js http://www.example.com --modules windowPerformance --verbose --gecko

* timeToFirstByte: 280 ms
* timeToLastByte: 283 ms
* domInteractive: 38 ms
* domContentLoaded: 35 ms
* domContentLoadedEnd: 54 ms
* domComplete: 59 ms
* timeBackend: 83 %
* timeFrontend: 17 %
* performanceTimingConnect: 0 ms
* performanceTimingDNS: 0 ms
* performanceTimingPageLoad: 39 ms
* performanceTimingTTFB: 0 ms
```
